src/neptune/internal/init/run.py

```diff
@@ -141,25 +143,25 @@ def init_run(
             Unix style pathname pattern expansion is supported. For example, you can pass `*.py` to upload
             all Python files from the current directory.
             If None is passed, the Python file from which the run was created will be uploaded.
-        capture_stdout: Whether to log the stdout of the run. Defaults to True.
+        TODO: capture_stdout: Whether to log the stdout of the run. Defaults to True.
```

Comment:
@normandy7 I need you here 🚀. After this PR we're going to log all monitoring metadata (CPU, GPU, memory usage, standard console output etc.) under `monitoring/<some-hash>` for instance `monitoring/ab8c2f0d`. This hash will be generated based on some environment information (network name of user computer, process identifier, thread identifier). I really have no idea how to rephrase it properly. Could you help me?

Reply:
Yes. I'll propose something.